cmd/podman/containers/cp.go

```diff
@@ -1,16 +1,25 @@
 package containers
 
 import (
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	buildahCopiah "github.com/containers/buildah/copier"
 	"github.com/containers/podman/v2/cmd/podman/common"
 	"github.com/containers/podman/v2/cmd/podman/registry"
+	"github.com/containers/podman/v2/pkg/copy"
 	"github.com/containers/podman/v2/pkg/domain/entities"
+	"github.com/containers/podman/v2/pkg/errorhandling"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
 var (
-	cpDescription = `Command copies the contents of SRC_PATH to the DEST_PATH.
+	cpDescription = `Copy the contents of SRC_PATH to the DEST_PATH.
 
-  You can copy from the container's file system to the local machine or the reverse, from the local filesystem to the container. If "-" is specified for either the SRC_PATH or DEST_PATH, you can also stream a tar archive from STDIN or to STDOUT. The CONTAINER can be a running or stopped container.  The SRC_PATH or DEST_PATH can be a file or directory.
+  You can copy from the container's file system to the local machine or the reverse, from the local filesystem to the container.  If "-" is specified for either the SRC_PATH or DEST_PATH, you can also stream a tar archive from STDIN or to STDOUT.  The CONTAINER can be a running or stopped container.  The SRC_PATH or DEST_PATH can be a file or a directory.
```

Comment:
It is no longer acceptable to double-space after periods. (It's a habit that has taken me a long time to break, but I accept the justification).

Reply:
I though this was just those of us that have actually used a typewriter.

Reply:
I am too old to change, @edsantiago

Reply:
> I thought this was just those of us that have actually used a typewriter.

I learned on a typewriter :)  I always use double spaces for text and now my brain parser is painfully dependent on it.